packages/compass-schema/src/components/export-schema-modal.tsx

```diff
@@ -178,10 +184,12 @@ const ExportSchemaModal: React.FunctionComponent<{
           Cancel
         </Button>
         <Button
-          onClick={() => {
-            /* TODO(COMPASS-8704): download and track with trackSchemaExported */
-          }}
           variant="primary"
+          isLoading={exportStatus === 'inprogress'}
+          loadingIndicator={<SpinLoader />}
+          disabled={!exportedSchema}
+          onClick={onSchemaDownload}
```

Comment:
Are we planning on adding a visual indicator that the file is saving & saved? Right now the save dialogue closes and they're back to the export page which they would then cancel out of. Looking at the designs there is an `in progress` toast included:
https://www.figma.com/design/CBHJriBQZ2qxMxtnXFgDxJ/EXPO-3174-%3A-Export-Schema-%2B-Validation-Detection?node-id=1298-55009&t=MBxcSDens4RhwtPM-1

Reply:
I think when we have toasts that show that the file was saved, closing the modal makes sense.